src/pytest_docker/__init__.py

```diff
@@ -103,9 +109,32 @@ def docker_compose_file():
 
 
 @pytest.fixture(scope='session')
-def docker_services(docker_compose_file):
+def allow_no_docker():
+    """Return if want to run against localhost when docker is not available.
+
+    Override this fixture to return `True` if you want the ability to
+    run without docker.
+
+    """
+    return False
+
+
+@pytest.fixture(scope='session')
+def docker_services(docker_compose_file, allow_no_docker):
     """Ensure all Docker-based services are up and running."""
 
+    # If we allowed to run without Docker, check it's presence
+    if allow_no_docker is True:
+        try:
+            execute('docker ps')
+        except:
```

Comment:
Should probably trap a specific exception and check the output (what happens if I CTRL-C at this point?)